### ISSUE TITLE
Creating a Teams blocked

### ISSUE THREAD

**Issue by AlexPopescu30** [OfficeDev/microsoft-teams-apps-requestateam#397]
### Documentation inaccuracy

Hello,

I'm posting this question here as I don't think it's a bug, rather a question for this app.

I have this scenario where, creating a Team is blocked for everyone from the tenant expect for a specific M365 Group. The members of this group are the global admins and the service account used for the Request-a-team app.

Is there an impact on the application level if this scenario is in place?

### Additional Info

I would help with more information is needed. thanks a lot

**Comment by jesusfer** [comment]
No, as long as the service account can create new teams, the application will still work.
